engine/src/flutter/lib/web_ui/dev/steps/run_suite_step.dart

```diff
@@ -173,13 +173,11 @@ class RunSuiteStep implements PipelineStep {
 
   Future<SkiaGoldClient?> _createSkiaClient() async {
     if (suite.testBundle.compileConfigs.length > 1) {
-      print('Not creating skia client due to multiple compile configs');
```

Comment:
These log statements were quite useful to look in CI and see if there were reasons we didn't create a gold client. I left them in because they actually aren't particularly spammy, they happen once per suite. Do we actually have to remove these? The idea is to give some information in stdout as to whether this suite is actually creating a gold client and successfully using it

Reply:
What I was trying to do here is keep the happy path as clean as possible. These cases (multiple configs, and safari) are well-known and expected to not create a Skia client, so I didn't see the value in printing this message over and over.

To me, a printed message is valuable when something unexpected happens, e.g. We couldn't create a Skia client on Chrome where we expect it to be created.

I do understand your POV though. If you feel strongly about it, I can put these messages back.